dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -216,6 +218,38 @@ destroy_ocp() {
     $OC_BINARY cluster down
 }
 
+remove_che_from_ocp() {
+	echo "[CHE] Checking if project \"${CHE_OPENSHIFT_PROJECT}\" exists before removing..."
+	WAIT_FOR_PROJECT_TO_DELETE=true
+	CHE_REMOVE_PROJECT=true
+	DELETE_OPENSHIFT_PROJECT_MESSAGE="[CHE] Removing Project \"${CHE_OPENSHIFT_PROJECT}\"."
+	if $OC_BINARY get project "${CHE_OPENSHIFT_PROJECT}" &> /dev/null; then
+		echo "[CHE] Project \"${CHE_OPENSHIFT_PROJECT}\" exists."
+		while $WAIT_FOR_PROJECT_TO_EXIT
```

Comment:
Where initial value assigned to this variable? Maybe `WAIT_FOR_PROJECT_TO_DELETE` should be here

Reply:
good catch! I guess `WAIT_FOR_PROJECT_TO_DELETE` should be renamed to `WAIT_FOR_PROJECT_TO_EXIT`

Reply:
@sleshchenko Thanks. Hard to test for problems like this. Fixed.